dashboard/analyse_dashboard/analyse/Analyse/DFN.py

```diff
@@ -0,0 +1,306 @@
+from Analyse.Data import Data
+from Analyse.FttX import FttXExtract, FttXTransform, FttXAnalyse, FttXETL, PickleExtract, FttXTestLoad
+from Analyse.Record import ListRecord, IntRecord, StringRecord, Record, DateRecord, DictRecord
+from functions import get_data_targets_init_dfn, error_check_FCBC, get_start_time, get_timeline, get_total_objects, \
+    prognose, targets, performance_matrix, prognose_graph, overview, graph_overview, \
+    info_table, analyse_documents, calculate_jaaroverzicht, preprocess_for_jaaroverzicht
+import pandas as pd
+
+import logging
+
+logger = logging.getLogger('DFN Analyse')
+
+
+class DFNExtract(FttXExtract):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.planning_location = kwargs['config'].get("planning_location")
+        self.target_location = kwargs['config'].get("target_location")
+
+    def extract(self):
+        self._extract_ftu()
+        self._extract_planning()
+        super().extract()
+
+    def _extract_ftu(self):
+        logger.info("Extracting FTU")
+        # doc = firestore.Client().collection('Data').document('analysis').get().to_dict()
+        doc = None
+        if doc is not None:
+            date_FTU0 = doc['FTU0']
+            date_FTU1 = doc['FTU1']
+        else:
+            logger.warning("Could not retrieve FTU0 and FTU1 from firestore, getting from local file")
+            date_FTU0, date_FTU1 = get_data_targets_init_dfn(self.target_location)
```

Comment:
Volgens mij kunnen we gewoon dezelfde functie gebruiken als voor kpn? dit houdt alles ook generieker

Reply:
Voorstel:
- kopieer de functie uit KPN class
- Pas de excel voor KPN zo aan dat die de dummy data van dfn reflecteert